common_plaintext_files/BUILD_DETAIL.md.erb

```diff
@@ -92,22 +92,23 @@ $ bundle exec yard doc --no-cache
 $ bin/yard doc --no-cache
 ```
 
-## Rubocop
+## RuboCop
 
-We use [Rubocop](https://github.com/bbatsov/rubocop) to enforce style conventions on the project so
-that the code has stylistic consistency throughout. Run with:
+We use [RuboCop](https://github.com/rubocop-hq/rubocop) to enforce style
+conventions on the project so that the code has stylistic consistency
+throughout. Run with:
 
 ```
-$ bundle exec rubocop lib
+$ bundle exec rubocop
```

Comment:
[`rspec-rails`](https://github.com/rspec/rspec-rails/pull/2272) and [`rspec-expectations`](https://github.com/rspec/rspec-expectations/pull/1104) are offence-free to my best memory, including `spec/`. `rspec-mocks` and `rspec-core` are not. Their `.rubocop.yml` files are different, and only `lib/` is offence-free.
Unfortunately, it is not consistent.

Reply:
Ok, we'll just stick to `lib` for all repos.